docs/data/base/components/slider/DiscreteSlider.js

```diff
@@ -0,0 +1,115 @@
+import * as React from 'react';
+import { styled, alpha, Box } from '@mui/system';
+import SliderUnstyled from '@mui/base/SliderUnstyled';
+
+const StyledSlider = styled(SliderUnstyled)(
```

Comment:
We should improve the hover styles:

![image](https://user-images.githubusercontent.com/4512430/158790065-46304595-c1ca-4949-87a3-227cce5ec854.png)

Also, we need a focus visible styles on the thumb, there is not indication that it is focused.